### PR TITLE
Add python API for appending LoD level

### PR DESCRIPTION
**Pull request by wanghaoshuang**
Make lod reset op support for append lod level

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -7060,8 +7061,60 @@ def lod_reset(x, y=None, target_lod=None):
...
+            level: [1, 1, 1, 1, 1, 1, 1]
+
+            then we get a 2-level LoDTensor:
+                x.lod =  [[ 2, 3, 1 ], [1, 1, 1, 1, 1, 1]]
+                x.data = [[1.0], [2.0], [3.0], [4.0], [5.0], [6.0]]
+                x.dims = [6, 1]
 
+    Args:
+        x (Variable): Input variable which could be a tensor or LoDTensor.
+        level (list|tuple): The LoD level to be appended into LoD of x..
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: 多了个.

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -7060,8 +7061,60 @@ def lod_reset(x, y=None, target_lod=None):
...
+
+    Returns:
+        Variable: Output variable with new LoD.
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: new LoD level ?

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (wanghaoshuang) [reply python/paddle/fluid/layers/nn.py]: Thx. Fixed.

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (wanghaoshuang) [reply python/paddle/fluid/layers/nn.py]: Thx. Fixed.

**Pull request merged**